Source/Scene/Scene.js

```diff
@@ -1752,9 +1801,16 @@ define([
 
         this._tweens.update();
         this._camera.update(this._mode);
-        this._screenSpaceCameraController.update();
+
+        if (!this._useWebVR || this._mode === SceneMode.SCENE2D) {
```

Comment:
>Did you test VR in CV?

Yes

>Why is it disabled in 2D?

Its not. Changing the camera view from the device orientation is disabled since there is always a top down view. We could disable it in 2D since VR doesn't work in 2D mode. Everything is on the same plane as the 2D map.